sdk/python/kfp/compiler_cli_tests/test_data/v2_component_with_pip_index_urls.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
I think we can remove this json golden here. Though you would need json instead of yaml in `sdk/release-1.8` branch.

Reply:
+1 here. Otherwise lgtm!